src/test/groovy/graphql/StarWarsQueryTest.groovy

```diff
@@ -190,7 +190,7 @@ class StarWarsQueryTest extends Specification {
                 ]
         ]
         when:
-        def result = GraphQL.newGraphQL(StarWarsSchema.starWarsSchema).build().execute(query, null, params).data
+        def result = GraphQL.newGraphQL(StarWarsSchema.starWarsSchema).build().execute(query, null, params).data // Retain deprecated method for test coverage
```

Comment:
Sorry to be a pain but I think we should "retain the deprecated method coverage IN the test for that class

ex `GraphQLTest` say rather than some random other test

This was the scope of the "retained for deprecated reasons" will reside in the most logical spot

Reply:
That's a good idea, makes it easier to find deprecated uses later